src/development/LocalTest/Services/Storage/Implementation/ApplicationRepository.cs

```diff
@@ -16,11 +17,16 @@ namespace LocalTest.Services.Storage.Implementation
     public class ApplicationRepository : IApplicationRepository
     {
         private readonly ILocalTestAppSelection _localTestAppSelectionService;
+        private readonly ILocalApp _localApp;
         private readonly ILogger _logger;
 
-        public ApplicationRepository(ILocalTestAppSelection localTestAppSelectionService, ILogger<ApplicationRepository> logger)
+        public ApplicationRepository(
+            ILocalTestAppSelection localTestAppSelectionService,
+            ILocalApp localApp,
+            ILogger<ApplicationRepository> logger)
         {
             _localTestAppSelectionService = localTestAppSelectionService;
```

Comment:
Cant find any references to _localTestAppSelectionService in the file, so it should be removed.

Reply:
The only remaining reference to `ILocalTestAppSelection` is actually in `TextRepository.cs`

 https://github.com/Altinn/altinn-studio/blob/88059f22d1926d7a5540ae04e16ad4b8cc370b61/src/development/LocalTest/Services/Storage/Implementation/TextRepository.cs#L52
 
 Seems to me like it is not actually in use, but I can't say that with confidence, as it is hard to search for places the api `storage/api/v1/applications/{org}/{app}/texts/{language}` might be in use.